Clipboard.xs

```diff
@@ -468,6 +468,34 @@ PPCODE:
 		XSRETURN_NO;
 	}
 
+void
+UGet()
+PPCODE:
+    HANDLE myhandle;
+    if(OpenClipboard(NULL)) {
+	EXTEND(SP,1);
+	if(myhandle = GetClipboardData(CF_UNICODETEXT)) {
+	    /* here we decode UTF16-LE into UTF8, using perl API */
+	    wchar_t *wcmyh = (wchar_t*)myhandle;
+            int i, len = wcslen(wcmyh);
+            SV *sv = newSV(len * UTF8_MAXBYTES +1);
+	    SvPOK_on(sv);
+            U8 *e = (U8*) SvPVX(sv), *e0 = e;
+	    for (i=0; i<len; i++) {
+		e = uvuni_to_utf8(e, wcmyh[i]);
+	    }
```

Comment:
This doesn't handle surrogate pairs at all and should be using [WideCharToMultiByte()](https://docs.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-widechartomultibyte) with CodePage set to CP_UTF8 to translate it.  Unfortunately perl's builtin functions to do this conversion are not API.